### ISSUE TITLE
SmokeDetectionSystem, Intrusion Detection System and PresenceSimulationService

### ISSUE THREAD

**Issue by TSK7M** [tschamm/boschshc-hass#137]
Hello, 
I have two Bosch SHC 1. 
Unfortunately I cannot distinguish between the bridges when configuring SmokeDetectionSystem, Intrusion Detection System and PresenceSimulationService.
Is there a way to distinguish between the SHCs (with two SHCs). 

Thanks and greetings

---------------------

Hallo, 
ich habe zwei Bosch SHC 1 ich kein leider bei der Konfiguration von SmokeDetectionSystem, Intrusion Detection System und PresenceSimulationService nicht zwischen den Bridges unterscheiden. 
Ich kann dann auch nicht festlegen was bei welcher Bridge ausgeführt werden soll, da ich nur eine Bridge sehe.

---------------------

Only one bridge is displayed:
<img width="821" alt="image" src="https://github.com/tschamm/boschshc-hass/assets/151848159/a96e3016-9b92-433b-8589-bceec876c10d">

<img width="506" alt="image" src="https://github.com/tschamm/boschshc-hass/assets/151848159/d1c7acb5-2c0d-4ac0-8a4f-6d1239b1da6d">

<img width="804" alt="image" src="https://github.com/tschamm/boschshc-hass/assets/151848159/3b13f22d-40bb-44c3-96fe-a15874cdff0a">

<img width="821" alt="image" src="https://github.com/tschamm/boschshc-hass/assets/151848159/a503f1aa-d527-47fc-98c7-c30f0765d02c">

<img width="508" alt="image" src="https://github.com/tschamm/boschshc-hass/assets/151848159/d3efc7fe-a951-4260-ab6a-4121c07602fd">

<img width="511" alt="image" src="https://github.com/tschamm/boschshc-hass/assets/151848159/d054438c-c43a-405c-a5d1-3e09ed1dbda9">

<img width="499" alt="image" src="https://github.com/tschamm/boschshc-hass/assets/151848159/dcc05d71-288d-4588-8d77-0881f625c388">

**Comment by tschamm** [comment]
That looks for me as if the unique IDs of the central services are somehow mixed. I encountered a similar behavior some months ago, but then it was sufficient to add the root device id (mac-address of SHC) to the unique id.
Can you show me the device description for both entities from the entity_registry (config/.storage/core.entity_registry)

**Comment by TSK7M** [comment]
@tschamm Yes, I can send the data but I don't want to make it public. Where can I provide the data?

**Comment by tschamm** [comment]
Fair, I think no sensitive data needs to be shared. You can obfuscate all that data, I just need to know whether some IDs are identical or differently.
You can send me a PM via the HA community forum (user is tschamm)

**Comment by TSK7M** [comment]
@tschamm I sent the link as a PM (HA community forum)

**Comment by mosandlt** [comment]
This is resolved in the 0.5.x line. Entity unique_ids were reworked to be prefixed with the controller's own device id (`{root_device_id}_{device.id}`), so the central singletons (Intrusion Detection, Smoke Detection, Presence Simulation) no longer collide when two SHC controllers are configured — each controller's instances now get distinct unique_ids.

Please update to **0.5.0+**. Closing — reopen if you still see duplicated central entities after updating.
